### DISCUSSION TITLE
problem for installing beis

### DISCUSSION THREAD

**Discussion by alirahdarian** [pybamm-team/PyBaMM#4061] [General]
Hello,

I am trying to install beis in Ubuntu (based on the instructions in https://github.com/rish31415/pybamm-eis) but when i enter the following  command: 
git clone https://github.com/rish31415/pybamm-eis
I receive the following error:
Cloning into 'pybamm-eis'...
fatal: unable to access 'https://github.com/rish31415/pybamm-eis/': Failed to connect to github.com port 443 after 131593 ms: Connection timed out

Does some know what should i do?

**Comment by agriyakhetarpal** [comment]
Hi, @alirahdarian – this looks like an issue stemming from your internet connection; please consult with your internet provider or retry with a different connection that is able to provide reliable access to Git repositories. This does not appear to be a PyBaMM-related issue, and I can clone the requested repository:

```bash
$ git clone https://github.com/rish31415/pybamm-eis
Cloning into 'pybamm-eis'...
remote: Enumerating objects: 683, done.
remote: Counting objects: 100% (287/287), done.
remote: Compressing objects: 100% (151/151), done.
remote: Total 683 (delta 184), reused 216 (delta 129), pack-reused 396
Receiving objects: 100% (683/683), 930.33 KiB | 4.82 MiB/s, done.
Resolving deltas: 100% (420/420), done.
```
